core/trino-parser/pom.xml

```diff
@@ -73,6 +73,14 @@
     </dependencies>
```

Comment:
so if you do `mvn install` what get installed into maven local repository? Is it part of the existing jar? How about attaching grammar file as separate artifact?

```
                    <plugin>
                        <groupId>org.codehaus.mojo</groupId>
                        <artifactId>build-helper-maven-plugin</artifactId>
                        <executions>
                            <execution>
                                <id>attach-artifacts</id>
                                <phase>package</phase>
                                <goals>
                                    <goal>attach-artifact</goal>
                                </goals>
                                <configuration>
                                    <artifacts>
                                        <artifact>
                                            <file>path/to/grammar.gr</file>
                                            <type>g4</type>
                                        </artifact>
                                    </artifacts>
                                </configuration>
                            </execution>
                        </executions>
                    </plugin>
```